test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
- [ ] Nice working on your tests, Kindly check all your tests are not passing see the screenshot below.
![image](https://user-images.githubusercontent.com/48371156/202874808-e9d990d8-2247-4af8-88c8-e51c93110eb1.png)

Reply:
Thank you @danielufeli  for noting when making the previous changes we forgot to push the new  SPEC test files,
We have made the commit now and we will make a resubmission.
<img width="846" alt="image" src="https://user-images.githubusercontent.com/106079814/202875082-d648d820-c835-477e-9483-8159fbf97d16.png">